diagnostic.md

```diff
@@ -177,5 +201,7 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
+It would stretch to 125px, because 25px is 25% of 100 and a screen size of 550px would be greater than 400 and less than 800, so the rules of the first media query would apply.
 
-<!-- your answer ends here -->
+I may be wrong in that the percentage means of the width of the window, so it may be just 100px, because that is 25% of the 400px window itself. I tried to look that up, but ran out of time. 
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have ```.stretch``` be 25% and at 550px width, 25% is 137.5 px.